src/components/EmailSubscription/index.view.tsx

```diff
@@ -70,13 +84,36 @@ const EmailSubscriptionForm: React.FC = () => {
           </div>
         </form>
       )}
-      {showFeedback && (
-        <div className="EmailSubscription__feedbackContainer">
-          <span className="EmailSubscription__feedbackMessage">
-            {feedbackMessage}
-          </span>
+      <div className="EmailSubscription__feedbackContainer">
+        <div className="EmailSubscription__icon">
+          {requestState === states.LOADING && (
+            <ClipLoader size={25} color={wedgewood} loading={true} />
+          )}
+          {requestState === states.SUBMITTED && (
+            <Icon
+              path={mdiCheckCircleOutline}
+              title="Successful Request"
+              size={1}
+              horizontal
+              vertical
+              color="green"
+            />
+          )}
+          {requestState === states.ERRORED && (
+            <Icon
+              path={mdiAlertCircle}
+              title="Error Occured"
+              size={1}
+              horizontal
+              vertical
+              color="red"
```

Comment:
Small suggestion - is it possible to not use pure red? On Chrome, the green looks fine but the red is a little harsh.

Reply:
To piggy back his comment, we can also get rid of the `horizontal` and `vertical` attributes for both Icons since it seems like it flips them. Should have paid a bit more attention to this before pushing these changes 
😬

Reply:
I can look into Adobe's color wheel to see if we can find a complimentary red and green with wedgewood!